hamilton/plugins/h_ray.py

```diff
@@ -135,6 +163,14 @@ def build_result(self, **outputs: typing.Dict[str, typing.Any]) -> typing.Any:
         result = ray.get(remote_combine)  # this materializes the object locally
         return result
 
+    def post_graph_execute(self, *args, **kwargs):
+        """We have the option to close the cluster down after execution."""
+
+        if not self.shutdown_ray_on_completion:
```

Comment:
I think we want this right?
```suggestion
        if self.shutdown_ray_on_completion:
```